tools/mo/openvino/tools/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -513,7 +513,7 @@ def serialize_network(graph, net_element, unsupported):
             continue
 
         # Here output data node count is checked. Each port cannot have more than one data node.
```

Comment:
Please correct this comment and note that there are cases of more than 1 data note.

Reply:
uups, missed that. I'll fix

Reply:
fixed that